xml/System.CommandLine.Builder/CommandLineBuilderExtensions.xml

```diff
@@ -0,0 +1,700 @@
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.CommandLine.Builder.CommandLineBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="System.CommandLine.Builder.CommandLineBuilder" RefType="this" />
+      </Parameters>
+      <Docs>
+        <param name="builder">A command line builder.</param>
+        <summary>
+            Enables signaling and handling of process termination via a <see cref="T:System.Threading.CancellationToken" /> that can be passed to a <see cref="T:System.CommandLine.Invocation.ICommandHandler" /> during invocation.
+            </summary>
+        <returns>The same instance of <see cref="T:System.CommandLine.Builder.CommandLineBuilder" />.</returns>
+        <remarks>To be added.</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="EnableDirectives">
+      <MemberSignature Language="C#" Value="public static System.CommandLine.Builder.CommandLineBuilder EnableDirectives (this System.CommandLine.Builder.CommandLineBuilder builder, bool value = true);" />
+      <MemberSignature Language="ILAsm" Value=".method public static hidebysig class System.CommandLine.Builder.CommandLineBuilder EnableDirectives(class System.CommandLine.Builder.CommandLineBuilder builder, bool value) cil managed" />
+      <MemberSignature Language="DocId" Value="M:System.CommandLine.Builder.CommandLineBuilderExtensions.EnableDirectives(System.CommandLine.Builder.CommandLineBuilder,System.Boolean)" />
+      <MemberSignature Language="VB.NET" Value="&lt;Extension()&gt;&#xA;Public Function EnableDirectives (builder As CommandLineBuilder, Optional value As Boolean = true) As CommandLineBuilder" />
+      <MemberSignature Language="F#" Value="static member EnableDirectives : System.CommandLine.Builder.CommandLineBuilder * bool -&gt; System.CommandLine.Builder.CommandLineBuilder" Usage="System.CommandLine.Builder.CommandLineBuilderExtensions.EnableDirectives (builder, value)" />
+      <MemberType>Method</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.CommandLine</AssemblyName>
+        <AssemblyVersion>2.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <ReturnValue>
+        <ReturnType>System.CommandLine.Builder.CommandLineBuilder</ReturnType>
+      </ReturnValue>
+      <Parameters>
+        <Parameter Name="builder" Type="System.CommandLine.Builder.CommandLineBuilder" RefType="this" />
+        <Parameter Name="value" Type="System.Boolean" />
+      </Parameters>
+      <Docs>
+        <param name="builder">A command line builder.</param>
+        <param name="value">If set to <see langword="true" />, then directives are enabled. Otherwise, they are parsed like any other token.</param>
```

Comment:
```suggestion
        <param name="value">
          <see langword="true" /> to enable directives. <see langword="false" /> to parse like any other token.</param>
```

Reply:
to parse like any other token --> to parse them like any other token

Reply:
I updated the wording to be more explicit.

Reply:
Okay, updated again.